code/src/components/ArtistDetails.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+
+export const ArtistDetails = (props) => {                //Display of the Artists names, added a link to their artist page
+    return (
+        <h3>{props.Artist.map((item, index) => {
+            if (props.Artist.length - 1 === index)
+            {
+                return <a className="Artist" key={item.name} href={item.external_urls.spotify} target="_blank">{item.name}</a> //
```

Comment:
i liked that you made the , and & happen by coding it directly in component instead of css for example